src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
> or we need to revise our rather carefully designed rules, or we drop namespaces.`


No reason to be so drastic, no reason to drop any rules ;-). Since what you say is not correct. 

> With our current agreement about how the UI framework should work, we can not have a wrapping using dividers. Components should come with their renderers for their own namespace, that just renders the components in that namespace and nothing else.

A renderer of a component is responsible to place all direct children in their according places, where they will render their own content (as you correctly point out). dividers and items are direct children of listings with both having their own renderer. There is really no issue with the list wrapping them to accordions. It's just a matter of where you think the list to begin and where you believe the divider to start. But that is open for discussion IMHO.

Let's make an example:
```
<div class="some-listing-as-accordion">
	<div class="wrapping-of-one-set-of-divider-with-item">
		<div class="panel-heading" role="tab" id="headingOne">
			<a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
				<div class="Rendering-Domain-of-Divider">Title of Divider</div>
			</a>
		</div>
		<div id="collapseOne" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
			<div class="panel-body">
				<div class="rendering-domain-of-item">items content</div>
			</div>
		</div>
	</div>
</div>
```

> Dividers do not allow accordions or we need to have listing dividers and dropdown dividers

I would rather attach a context specific renderer.